### PR TITLE
Feature/layout: implemented desktop layout

### PR DESCRIPTION
**Pull request by petrovskya**
1. feat: implemeted Helyfly desktop layout
2. Changes:
    - added layout to eight sections of the page;
    - added CSS styles to each section;
    - added global styles, variales & mixins for styles;
    - added readme file for the project;
3. How to test:
    - <a href="https://petrovskya.github.io/Helyfly/">View Demo</a>
    - <a href="https://www.figma.com/file/fMyOAiBkWJ4LtAacwdmIyB/testTask?node-id=0%3A1&t=vpEqZ4WXzU4tIKIF-0">View layout in Figma</a>

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+  background-image: url("../assets/backgrounds/headerBannerBg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+.header-banner__title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 49px;
+  color: #ffffff;
```
Comment (irina-komissarova) [inline styles/styles.css]:
```suggestion
  color: #fff;
```

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+  color: #5286ec;
+}
+.header-banner {
+  padding: 286px 0 260px;
+  background-image: url("../assets/backgrounds/headerBannerBg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
+.header-banner__title {
```
Comment (irina-komissarova) [inline styles/styles.css]: please add empty lines between the groups of properties all over the project

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 49px;
+  color: #ffffff;
+}
+.header-banner__button {
+  position: relative;
+  margin-top: 129px;
+  padding: 20px 38px 20px;
```
Comment (irina-komissarova) [inline styles/styles.css]:
```suggestion
  padding: 20px 38px;
```

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,627 @@
...
+  border: 3px solid #ffffff;
```
Comment (irina-komissarova) [inline styles/styles.css]: please fix according to previous comments

File: styles/styles.css [hunk 5]
```diff
@@ -0,0 +1,627 @@
...
+  border-radius: 3px;
+  font-size: 18px;
+  font-weight: 500;
+  font-weight: 900;
```
Comment (irina-komissarova) [inline styles/styles.css]: !

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,627 @@
...
+  color: #ffffff;
```
Comment (irina-komissarova) [inline styles/styles.css]: same

File: styles/styles.css [hunk 7]
```diff
@@ -0,0 +1,627 @@
...
+.about__article_show-more {
+  color: #5286ec;
+}
+.about__card {
+  display: flex;
+  align-items: flex-start;
+  max-width: 50%;
+  margin: 0 15px;
+  padding-top: 31px;
+  padding-right: 113px;
```
Comment (irina-komissarova) [inline styles/styles.css]: please combine it in one line

File: styles/styles.css [hunk 8]
```diff
@@ -0,0 +1,627 @@
...
+  align-items: center;
+  padding: 10px 0;
+  border-top: 0.5px solid #c7c7c7;
+}
+.booking__option-details {
+  display: grid;
+  gap: 10px;
+}
+.booking__option_time, .booking__option_other {
+  font-weight: 400;
```
Comment (irina-komissarova) [inline styles/styles.css]: it's a default value, could we get rid of it?

File: styles/styles.css [hunk 9]
```diff
@@ -0,0 +1,627 @@
...
+}
+.booking__option_price {
+  font-weight: 600;
+}
+.booking__option_button {
+  padding: 16px 32px;
+  border: 2px solid #fc9b09;
+  border-radius: 3px;
+  font-size: 14px;
+  font-weight: 400;
```
Comment (irina-komissarova) [inline styles/styles.css]: same, default value

File: styles/styles.css [hunk 10]
```diff
@@ -0,0 +1,627 @@
...
+  line-height: 17px;
+  color: #fc9b09;
+  transition: 0.3s;
+}
+.booking__option_button:hover {
+  background-color: #fc9b09;
+  color: #ffffff;
```
Comment (irina-komissarova) [inline styles/styles.css]:
```suggestion
  color: #fff;
```

File: styles/styles.css [hunk 11]
```diff
@@ -0,0 +1,627 @@
...
+}
+
+.location {
+  background: url("../assets/backgrounds/locationBg.png");
+  background-repeat: no-repeat;
+  background-size: cover;
```
Comment (irina-komissarova) [inline styles/styles.css]: could be combined in one property

File: styles/styles.css [hunk 12]
```diff
@@ -0,0 +1,627 @@
...
+  color: #37373f;
+  padding: 20px 0;
+}
+.location__info {
+  display: grid;
+  max-width: 420px;
+  padding: 51px 30px;
+  background-color: #ffffff;
```
Comment (irina-komissarova) [inline styles/styles.css]: !

File: styles/styles.css [hunk 13]
```diff
@@ -0,0 +1,627 @@
...
+}
+.location__slider {
+  counter-reset: myCounter;
+  font-size: 18px;
+}
+.location__slider-item {
+  list-style: none;
```
Comment (irina-komissarova) [inline styles/styles.css]: please fix properties order here and check all over styles

File: styles/styles.css [hunk 14]
```diff
@@ -0,0 +1,627 @@
...
+}
+.location__slider-item_text {
+  display: grid;
+  gap: 12px;
+}
+.location__slider-item_title {
+  font-weight: 500;
+}
+.location__slider-item_subtitle {
+  font-weight: 400;
```
Comment (irina-komissarova) [inline styles/styles.css]: default, could it be removed?

File: styles/styles.css [hunk 15]
```diff
@@ -0,0 +1,627 @@
...
+  align-items: center;
+  margin-right: 18px;
+  width: 61px;
+  height: 61px;
+  background: #fc9b09;
+  border-radius: 50%;
+  text-align: center;
+  font-size: 35px;
+  font-weight: 700;
+  color: #ffffff;
```
Comment (irina-komissarova) [inline styles/styles.css]: !

File: styles/styles.css [hunk 16]
```diff
@@ -0,0 +1,627 @@
...
+  line-height: 61px;
+}
+
+.comments {
+  padding: 140px 0 85px 0;
```
Comment (irina-komissarova) [inline styles/styles.css]:
```suggestion
  padding: 140px 0 85px;
```

File: styles/styles.css [hunk 17]
```diff
@@ -0,0 +1,627 @@
...
+  transform: translate(-50%, -50%);
+  object-fit: cover;
+}
+.comments__username {
+  margin-bottom: 10px;
+}
+.comments__user-position {
+  margin-bottom: 10px;
+  font-size: 14px;
+  font-weight: 400;
```
Comment (irina-komissarova) [inline styles/styles.css:446]: please check it it's necessary

File: styles/styles.css [hunk 18]
```diff
@@ -0,0 +1,627 @@
...
+  margin-bottom: 26px;
+}
+.comments__rate_score {
+  width: 118px;
+  height: 17px;
+}
+.comments__rate_status {
+  position: relative;
+  font-size: 14px;
+  font-weight: 400;
```
Comment (irina-komissarova) [inline styles/styles.css:463]: !

File: styles/styles.css [hunk 19]
```diff
@@ -0,0 +1,627 @@
...
+  margin-bottom: 22px;
+  font-size: 16px;
+  font-weight: 400;
+  text-align: left;
+  line-height: 24px;
+}
+.comments__date {
+  margin-bottom: 35px;
+  font-size: 14px;
+  font-weight: 400;
```
Comment (irina-komissarova) [inline styles/styles.css:486]: !

File: styles/styles.css [hunk 20]
```diff
@@ -0,0 +1,627 @@
...
+  position: relative;
+  font-size: 18px;
+  font-weight: 500;
+  color: #5286ec;
+}
+.comments__dropdown-button::before {
+  content: url("../assets/icons/arrowIcon.svg");
+  display: block;
+  position: absolute;
+  top: 0px;
```
Comment (irina-komissarova) [inline styles/styles.css]: please don't use px or any other units with 0

File: styles/styles.css [hunk 21]
```diff
@@ -0,0 +1,627 @@
...
+  left: -22px;
+}
+.comments__dropdown-button::after {
+  content: url("../assets/icons/arrowIcon.svg");
+  display: block;
+  position: absolute;
+  top: 0px;
```
Comment (irina-komissarova) [inline styles/styles.css]: !

File: styles/styles.css [hunk 22]
```diff
@@ -0,0 +1,627 @@
...
+  left: 175px;
+}
+
+.footer {
+  font-size: 14px;
+  font-weight: 400;
```
Comment (irina-komissarova) [inline styles/styles.css]: !

File: styles/styles.css [hunk 23]
```diff
@@ -0,0 +1,627 @@
...
+}
+.footer__info {
+  display: flex;
+  justify-content: space-between;
+  padding: 0px 66px 41px 66px;
```
Comment (irina-komissarova) [inline styles/styles.css]: !

Comment (irina-komissarova) [reply styles/styles.css]: also it can be simplified

File: styles/styles.css [hunk 24]
```diff
@@ -0,0 +1,627 @@
...
+}
+.footer__info_socials {
+  display: flex;
+  flex-direction: row;
```
Comment (irina-komissarova) [inline styles/styles.css]: default value, please remove

File: styles/styles.css [hunk 25]
```diff
@@ -0,0 +1,627 @@
...
+}
+.footer__lang-select {
+  color: #37373f;
+}
+.footer__wrapper {
+  background-color: #faf9f8;
+}
+.footer__other {
+  display: flex;
+  flex-direction: row;
```
Comment (irina-komissarova) [inline styles/styles.css]: !

File: styles/styles.css [hunk 26]
```diff
@@ -0,0 +1,627 @@
...
+  align-items: center;
+  padding: 3px 66px 31px 66px;
```
Comment (irina-komissarova) [inline styles/styles.css]: can be simplified as in previous comments

File: styles/styles.css [hunk 27]
```diff
@@ -0,0 +1,627 @@
...
+  color: #37373f;
+}
+.rate_features {
+  max-width: 569px;
+  padding: 20px 110px 25px 0;
+  border-bottom: 0.5px solid #c7c7c7;
+}
+.rate_booking {
+  max-width: 440px;
+  padding: 60px 0 50px 0;
```
Comment (irina-komissarova) [inline styles/styles.css]: simplify

File: styles/styles.css [hunk 28]
```diff
@@ -0,0 +1,627 @@
...
+}
+.profile-more_location {
+  text-align: left;
+}
+.profile-more_about {
+  width: 200px;
+  text-align: center;
+}
+
+/*# sourceMappingURL=styles.css.map */
```
Comment (irina-komissarova) [inline styles/styles.css]: ?

**Review by irina-komissarova — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 17 above)
Comment (petrovskya) [reply styles/styles.css:446]: The value from the layout for this element differs from the inherited
image

File: styles/styles.css (hunk 18 above)
Comment (petrovskya) [reply styles/styles.css:463]: The value from the layout for this element differs from the inherited. 
image

File: styles/styles.css (hunk 19 above)
Comment (petrovskya) [reply styles/styles.css:486]: The value from the layout for this element differs from the inherited.

**Review by irina-komissarova — APPROVED** [review]

**Pull request merged**